src/byui/cit260/sevenyearsofplenty/model/Crops.java

```diff
@@ -26,7 +26,8 @@
     private int pharoahsShare;
     private int fed;
     private int planted;
-    private ArrayList<Game> games = new ArrayList<Game>();
+    private int starvedPeople;
```

Comment:
Do we need to add a "starvedPeople" attribute? Can't this be internal to the control method and handled between "population" and "fed"?